cmd/mr_edit.go

```diff
@@ -22,36 +22,27 @@ var mrEditCmd = &cobra.Command{
 	Long:    ``,
 	Example: `lab MR edit <id>                                # update MR via $EDITOR
 lab MR update <id>                              # same as above
+lab MR update my-branch                         # same, but get MR ID from local branch
```

Comment:
Nitpick -- maybe "my-branch-name" ?  Only change this if I find something else to complain about :)

Reply:
I would even suggest `<branch-name>`, but I know there are other help messages around that don't make use of `<>`, so... :))

Reply:
Right, that would be more consistent with the `<id>` bit right above.